SpatialGDK/Source/Private/Interop/SpatialTypebindingManager.cpp

```diff
@@ -97,69 +100,85 @@ void USpatialTypebindingManager::CreateTypebindings()
 				}
 			}
 		}
-		
-		Info.SingleClientComponent = SchemaDatabase->ClassToSchema[Class].SingleClientRepData;
-		ComponentToClassMap.Add(Info.SingleClientComponent, Class);
 
-		Info.MultiClientComponent = SchemaDatabase->ClassToSchema[Class].MultiClientRepData;
-		ComponentToClassMap.Add(Info.MultiClientComponent, Class);
+		ForAllSchemaComponentTypes([&](ESchemaComponentType Type)
+		{
+			Worker_ComponentId ComponentId = SchemaDatabase->ClassToSchema[Class].SchemaComponents[Type];
+			if (ComponentId != 0)
+			{
+				Info.SchemaComponents[Type] = ComponentId;
+				ComponentToClassMap.Add(ComponentId, Class);
+				ComponentToOffsetMap.Add(ComponentId, 0);
+				ComponentToCategoryMap.Add(ComponentId, (ESchemaComponentType)Type);
+			}
+		});
 
-		Info.HandoverComponent = SchemaDatabase->ClassToSchema[Class].HandoverData;
-		ComponentToClassMap.Add(Info.HandoverComponent, Class);
+		Info.Class = Class;
 
-		Info.RPCComponents[RPC_Client] = SchemaDatabase->ClassToSchema[Class].ClientRPCs;
-		ComponentToClassMap.Add(Info.RPCComponents[RPC_Client], Class);
+		ClassInfoMap.Emplace(Class, Info);
+	}
 
-		Info.RPCComponents[RPC_Server] = SchemaDatabase->ClassToSchema[Class].ServerRPCs;
-		ComponentToClassMap.Add(Info.RPCComponents[RPC_Server], Class);
+	for (UClass* Class : SupportedClasses)
+	{
+		for (auto& SubobjectDataPair : SchemaDatabase->ClassToSchema[Class].SubobjectData)
+		{
+			int32 Offset = SubobjectDataPair.Key;
+			FSubobjectSchemaData SubobjectSchemaData = SubobjectDataPair.Value;
 
-		Info.RPCComponents[RPC_NetMulticast] = SchemaDatabase->ClassToSchema[Class].NetMulticastRPCs;
-		ComponentToClassMap.Add(Info.RPCComponents[RPC_NetMulticast], Class);
+			FClassInfo* ActorInfo = FindClassInfoByClass(Class);
+			FClassInfo* SubobjectInfo = FindClassInfoByClass(SubobjectSchemaData.Class);
```

Comment:
We should create a copy of this `FClassInfo` on stack, fill it out and then pass into `ActorInfo->SubobjectInfo`.